Update Necrosis.lua

RuskeMiahahn opened the pull request:
Demonic Sacrifice no longer costs mana, this check breaks the addon, meaning this check no longer makes sense and breaks the addon.

I have removed the mana check.

This change resolves issue #7

RuskeMiahahn committed:
Update Necrosis.lua

Demonic Sacrifice no longer costs mana, this check breaks the addon, meaning this check no longer makes sense and breaks the addon.

I have removed the mana check.

T321us commented:
If the spell no longer costs mana, removing the mana check is valid. But shouldn't the pet check remain?

RuskeMiahahn commented:
Considered that, but the whole section is based around mana checks, for all the spells above it in the block. Considering all this does is light up / grey out the button, felt cleaner to just remove it altogether.

However, if you'd prefer, feel free to just remove the "(NECROSIS_SPELL_TABLE[44].Mana > mana) or" instead and edit my pull request.

RuskeMiahahn committed:
Update Necrosis.lua

Returned Pet Check for Demonic Sacrifice check to show greyed out button when no pet summoned as per T321us request.

RuskeMiahahn commented:
I've updated my local version to test the no pet check staying in place, the button now appears greyed out in-game when I don't have a pet. The button still works regardless, this was a cosmetic feature that is currently fully breaking the addon for anyone with demonic sacrifice.

I've also updated the character encoding for the comments.

Please consider approving the changes so people can use the addon again.

RuskeMiahahn commented:
Is this repo still being maintained? Pull request to fix this bug was submitted 3 months ago.
